### DISCUSSION TITLE
Apple privacy manifest file

### DISCUSSION THREAD

**Discussion by yamade** [software-mansion/react-native-reanimated#5650] [Q&A] [answered]
Hello

You guys may notice this announcement from Apple.
https://developer.apple.com/support/third-party-SDK-requirements/

I think react-native-reanimated is using systemUptime.
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278394

So I'm wondering if you guys have any plan to add a privacy manifest file sometime soon. Thank you!

**Comment by angusryer** [comment, marked as answer]
Yes it seems to be used in older versions of `apple/sensor/ReanimatedSensor.m`

But PR #5819 appears to have replaced it, and is part of release 3.9.0-rc.0
